### ISSUE TITLE
AI-generated images on 404 page

### ISSUE THREAD

**Issue by lmcnulty** [responsible-ai-collaborative/aiid#1343]
We've floated the idea of using AI-generated images in parts of the site. A relatively harmless place we could test such a feature would be the 404 page. Sometimes I type https://incidentdatabase.ai/edit instead of https://incidentdatabase.ai/cite/edit. It could be fun to find the most common 404ing requests and use images generated by providing the 404ing url. I still don't have DALL-E access because I don't want to give OpenAI my phone number, but here's what I got from https://huggingface.co/spaces/stabilityai/stable-diffusion:

<details>
<summary>Output, not particularly creepy, no human faces</summary>

![image](https://user-images.githubusercontent.com/25443411/199755035-5de017ff-85cd-4955-aae0-eeab18429ec9.png)

</details>

**Comment by smcgregor** [comment]
We could manually select these, but I don't want to hook it into programmatically generated images based on the specific URL. People could share: incidentdatabase.ai/SUPER_OFFENSIVE_THING and generate an image we would then display...

**Comment by lmcnulty** [comment]
Yeah, that's why I suggested using the most commonly requested non-existent urls. Although I suspect that the top one will be something like `/wp-admin`...

**Comment by smcgregor** [comment]
Ha, bots hitting our non-existent wp admin page will be treated to very fanciful images indeed.

I think contents for one 404 page are good.
